src/ECER.Clients.Api/References/ReferenceEndpoints.cs

```diff
@@ -0,0 +1,33 @@
+﻿using ECER.Managers.Admin.Contract.References;
+using ECER.Utilities.Hosting;
+using MediatR;
+using Microsoft.AspNetCore.Http.HttpResults;
+
+namespace ECER.Clients.Api.References;
+
+public class ReferenceEndpoints : IRegisterEndpoints
+{
+  public void Register(IEndpointRouteBuilder endpointRouteBuilder)
+  {
+    endpointRouteBuilder.MapPost("/api/references", async Task<Results<Ok<GenerateVerificationLinkResponse>, BadRequest<string>>> (
```

Comment:
can you rename to /api/invitelinks? In general the feature is not only limited to references but to email invites in general